dronekit/__init__.py

```diff
@@ -2748,6 +2794,65 @@ def __setitem__(self, index, value):
         self._vehicle._wpts_dirty = True
 
 
+class ServoOutputRaw(object):
+    """
+    The RAW servo readings from the OUTPUT of the autopilot
+    This contains the true raw values without any scaling to allow data capture and system debugging.
+
+    The message definition is here: https://pixhawk.ethz.ch/mavlink/#SERVO_OUTPUT_RAW
+
+
+    """
+
+    def __init__(self,
```

Comment:
thinking about it kwargs makes sense here, if you don't have all the servo messages in kwargs, you won't be able to set them on the internal attrs of the class, you would then need to create a helper perhaps to get the servo message you are looking for `.get(1)` or some sort of helper that makes sense (this is just an idea)